### ISSUE TITLE
iP project formatting

### ISSUE THREAD

**Issue by shxr3f** [nus-cs2113-AY2122S2/forum#15]
Is it necessary for us to follow the formatting given by the website when implementing certain features for our iP.
For example, "deadline return book /by Sunday" this is the example input given on the website, is it okay if we decide to use other ways of implementing the same feature without the use of "/by".

**Comment by okkhoy** [comment]
Yes. That should be fine. But please document it in the user guide when you write it. (people should still be able to use/test your application)
